### PR TITLE
Replace use of deprecated methods with their recommended replacements

### PR DESCRIPTION
**Pull request by alsutton**
A couple of methods of moved to the commons-lang project, so here's a PR to get them from their new home.

### CODE REVIEW

**Comment by alsutton** [comment]
Yup, sorry for the noise. I've pushed a fix commit which adds the `java.scripting` jmod to proguards inputs so that it has the class information.

File: brut.apktool/apktool-cli/build.gradle [hunk 1]
```diff
@@ -58,6 +58,7 @@ task proguard(type: ProGuardTask, dependsOn: shadowJar) {
         libraryjars "${System.properties['java.home']}/jmods/java.xml.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
         libraryjars "${System.properties['java.home']}/jmods/java.desktop.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
         libraryjars "${System.properties['java.home']}/jmods/java.sql.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
+        libraryjars "${System.properties['java.home']}/jmods/java.scripting.jmod", jarfilter: '!**.jar', filter: '!module-info.class'
```
Comment (pashamcr) [inline brut.apktool/apktool-cli/build.gradle:61]: delete 58,59,60,61
add 58
`        libraryjars "${System.properties['java.home']}/jmods"`

Comment (alsutton) [reply brut.apktool/apktool-cli/build.gradle:61]: Are you sure you want to do two changes in one PR? (deprecation resolution and changing from a limited set of jmod references to all and any). I can see a potential issue with folk who may have locally customised their jmod definitions getting a different set of results to folk with an unmodified Java install.

**Review by iBotPeaches — APPROVED** [review]

**Comment by iBotPeaches** [comment]
thanks!

**Pull request merged**